catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'openedx-app-test'
+  description: "Test scripts for the Open edX mobile applications"
+  links:
+    - url: "https://github.com/openedx/openedx-app-test"
+      title: "openedx/openedx-app-test"
+      icon: "GitHub"
+  annotations:
+    openedx.org/arch-interest-groups: ""
+    openedx.org/release: "main"
+spec:
+  owner: group:openedx-unmaintained
```

Comment:
@volodymyr-chekyrta I think this should be owned by the mobile maintainers is that right?

Reply:
Just checked, we are using `group:openedx-mobile-maintainers` for mobile repos
https://github.com/openedx/openedx-app-ios/blob/main/catalog-info.yaml#L16